src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -395,6 +398,39 @@ public function phpinfoAction()
     }
 
     /**
+     * Displays the source of a file.
+     *
+     * @return Response A Response instance
+     *
+     * @throws NotFoundHttpException
+     */
+    public function openAction(Request $request)
+    {
+        if (null === $this->baseDir) {
+            throw new NotFoundHttpException('The base dir should be set.');
```

Comment:
Should be much more explicit than that with a bit of context on how to fix this.